### PR TITLE
Update packages, pull in compiled-only repos

### PR DESCRIPTION
**Pull request by jacogr**
... as a bonus, re-worked dapp methods permissions manager is available

cc @amaurymartiny

### CODE REVIEW

File: js/package.json [hunk 1]
```diff
@@ -44,6 +44,19 @@
     "test:coverage": "cross-env NODE_ENV=test istanbul cover _mocha -- --compilers ejs:ejsify 'src/**/*.spec.js'"
   },
   "devDependencies": {
+    "@parity/dapp-console": "paritytech/dapp-console",
```
Comment (tomusdrw) [inline js/package.json]: Was this change intended? Why is it oly devDependency?

Comment (jacogr) [reply js/package.json]: This is to make the distinction between -

(a) packages referenced in the actual code via `import ... from '...'` (dependency)
(b) packages only referenced in the webpack build (devDependency) - the dapps (mostly) fall into this category
